chatbot/setup_server.sh

```diff
@@ -33,13 +33,13 @@ else
     ufw allow https
     ufw enable
 
-    apt-get update
-    apt-get install software-properties-common
+    apt update
+    apt install software-properties-common
     add-apt-repository -y ppa:certbot/certbot
-    apt-get update
-    apt-get install -y nodejs npm certbot postgresql postgresql-contrib
+    apt update
+    apt install -y nodejs npm certbot postgresql postgresql-contrib
     npm install -g pm2 n
-    n stable
+    n 12.18.3         # keep this in sync with .nvmrc for Travis and with heartbeat/setup_server.sh
     npm install
```

Comment:
@schwarrrtz Just wondering if there's a reason that the `npm install` command is in the different place in these two install scripts?

Reply:
I'll move `npm install` in the other file to the bottom of this section to match the chatbot.

Reply:
don't think there's a reason.